client/src/components/AddCard/AddCardInfo/AddCardInfo.tsx

```diff
@@ -0,0 +1,24 @@
+import Typography from '@material-ui/core/Typography';
+import { Elements } from '@stripe/react-stripe-js';
+import { loadStripe } from '@stripe/stripe-js';
+import AddCardInfoForm from './AddCardInfoForm/AddCardInfoForm';
+
+const AddCardInfo = (props: any): JSX.Element => {
+  const options = {
+    clientSecret: props.location.state.clientSecret,
+    appearance: {},
```

Comment:
Question: What is the purpose of this field, I don't see us using it anywhere?

Reply:
I was not sure should I edit the Stripe components. Thank you for telling me it. Done.